main.go

```diff
@@ -17,3 +29,19 @@ func main() {
 	t := tree.Tree(lines, "/")
 	t.Print(0, nil)
 }
+
+func handleArguments() {
```

Comment:
Oh, I see. Yeah, it does handle `--`, but indeed does not have a concept of short vs. long flags.